dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/hooks/EnrollmentDateValidationHookTest.java

```diff
@@ -111,7 +111,7 @@ void testDatesMustNotBeInTheFuture()
         hasTrackerError( reporter, E1021, ENROLLMENT, enrollment.getUid() );
     }
 
-    @Test
+    // @Test
```

Comment:
?

Reply:
Ha! It's a test that fails when you're a developer testing your code and you're 5 hours west of UTC and it's within 5 hours of midnight, which frequently happens for me. I always have to disable this test to get all the other tests to run, and except for this time, I always remember to put it back. It's on my personal list to fix when my 2.38 features are done, since I am well situated ;) to test the fix.